core/common/src/DisplayStyleSettings.ts

```diff
@@ -78,6 +78,13 @@ export interface OrbitGtBlobProps {
   accountName: string;
 }
 
+/** JSON representation of the attribution properties for a reality model. These are used to create the logo card for the model using [IModelApp.makeLogoCard]($frontend).
+ * @beta
+ */
+export interface RealityModelAttribution {

```

Comment:
Might some people want to supply an icon as supported by `makeLogoCard`? Perhaps this and the argument to `makeLogoCard` should simply be called `LogoCardProps`?

Reply:
Yes - but Is there a way to store an icon as JSON?

Reply:
Only if it's a string that refers to a resource bundled with the app - so not usefully, no.